.github/workflows/publish-package.yml

```diff
@@ -74,6 +74,27 @@ jobs:
         name: python-package-distributions
         path: dist/
 
+    - uses: pnpm/action-setup@v2
+      with:
+        version: 9
```

Comment:
because we're specifying the pnpm version in the root package.json we have to remove the version number here, otherwise the pipeline will fail